azurerm/internal/services/hdinsight/hdinsight_hadoop_cluster_resource.go

```diff
@@ -34,14 +34,13 @@ var hdInsightHadoopClusterHeadNodeDefinition = azure.HDInsightNodeDefinition{
 var hdInsightHadoopClusterWorkerNodeDefinition = azure.HDInsightNodeDefinition{
 	CanSpecifyInstanceCount: true,
 	MinInstanceCount:        1,
-	MaxInstanceCount:        25,
 	CanSpecifyDisks:         false,
 }
 
 var hdInsightHadoopClusterZookeeperNodeDefinition = azure.HDInsightNodeDefinition{
 	CanSpecifyInstanceCount:  false,
 	MinInstanceCount:         3,
-	MaxInstanceCount:         3,
+	MaxInstanceCount:         utils.Int(2),
```

Comment:
Was this intended to be changed from 3 -> 2 ?

Reply:
This is by mistake, thank you for pointing this!